server/http4s-server/src/main/scala/sttp/tapir/server/http4s/TapirHttp4sServer.scala

```diff
@@ -12,6 +13,30 @@ import sttp.tapir.typelevel.ReplaceFirstInTuple
 import scala.reflect.ClassTag
 
 trait TapirHttp4sServer {
+  implicit class RichHttp4sHttpEndpoint0[I, E, O, F[_], G[_]](e: Endpoint[I, E, O, EntityBody[F]]) {
+    def toHttp(logic: I => G[Either[E, O]])(
+        t: F ~> G
```

Comment:
I feel like most of the time the expression for `t` would be shorter than one for `logic`. so I think we should swap the parameter lists. E.g. this:
```scala
endpoint.toHttp(ReaderT.liftK[F, Foo]) { i =>
  ReaderT { foo =>
    doSomething(i, foo)
  }
}
```
makes more sense to me than this:
```scala
endpoint.toHttp { i =>
  ReaderT { (foo: Foo) =>
    doSomething(i, foo)
  }
}(ReaderT.liftK[F, Foo])
```
, since by the time I get to `logic`, I can see what monad I'm looking at.